src/components/List/List.js

```diff
@@ -64,6 +68,19 @@ export default class List extends React.Component {
     </FlexView>
   );
 
+  templateError = (error) =>(
+    <FlexView
+      column
+      className="error"
+      vAlignContent='top'
+      hAlignContent='center'
+    >
+      <h1>Ops! We got an error {error.error.status}</h1>
+      <h3>{error.error.data.message}</h3>
```

Comment:
as always, cleaning up `template`s as much as possible is good: these could be two locals, e.g.
```js
{
  // ...
  errorStatus: error.error.status,
  errorMessage: error.error.data.message
  // ...
}
```